### PR TITLE
Create frontend-road-map.md

### PR DESCRIPTION
**Pull request by orlantquijada**

### CODE REVIEW

File: frontend-road-map.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+1. HTML/CSS
```
Comment (amoresjan) [inline frontend-road-map.md]: ok sir

Comment (citReyJoshua) [reply frontend-road-map.md]: Are you sure?

Comment (amoresjan) [reply frontend-road-map.md]: yes

Comment (noaypunk) [reply frontend-road-map.md]: boss

**Pull request merged**